services/viva-ms/lambdas/generateRecurringCaseHtml.js

```diff
@@ -89,7 +40,7 @@ export async function main(event, context) {
     return false;
   }
 
-  const [getClosedCasesError, { Items: closedCases }] = await getClosedUserCases(caseKeys.PK);
+  const [getClosedCasesError, { Items: closedCases }] = await to(getClosedUserCases(caseKeys.PK));
   if (getClosedCasesError) {
     log.error(
       'Error getting previous items from the cases table',
```

Comment:
Missing some logging arguments here?

Reply:
What do you mean?

Reply:
Never mind this. As you pointed out, the logging rutine needs some love.